pinot-common/src/main/java/org/apache/pinot/common/restlet/resources/StartReplaceSegmentsRequest.java

```diff
@@ -39,6 +40,12 @@ public class StartReplaceSegmentsRequest {
   private final List<String> _segmentsTo;
   private final Map<String, String> _customMap;
 
+  public StartReplaceSegmentsRequest(@JsonProperty("segmentsFrom") @Nullable List<String> segmentsFrom,
+      @JsonProperty("segmentsTo") @Nullable List<String> segmentsTo) {
+    this(segmentsFrom, segmentsTo, null);
+  }
+
+  @JsonCreator
   public StartReplaceSegmentsRequest(@JsonProperty("segmentsFrom") @Nullable List<String> segmentsFrom,
```

Comment:
looks like the existing constructor already accepts all parameters required by the new one you added. why not just use this existing constructor?

Reply:
This change actually breaks [current usage of the API](https://github.com/apache/pinot/pull/10679/files), by adding the new parameter.  Any user who pulls in this code, will see a compilation failure/build break. It used to take only 2 params and now the user has to pass in a null.

Reply:
understood. thx for the context.